src/Cargo.lock

```diff
@@ -2237,6 +2237,7 @@ dependencies = [
 name = "rustc_privacy"
 version = "0.0.0"
 dependencies = [
+ "log 0.4.3 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Could you avoid introducing this new dependency?

Reply:
Any `debug!("...")` calls would then also need to be removed. Is this version of log different from that used by the rest of the rustc_* crates?

Reply:
`debug!(...)` logging used for a specific debugging session is usually useless for anything else in the future, so I mean yes, it is better removed.